docs/INSTALL-email.md

```diff
@@ -35,7 +35,7 @@ Go to [My Account page](https://app.mailjet.com/account) and click on the ["SMTP
     SMTP password?       [Elastic Email API Key]
     SMTP port?           2525
     
-NOTE: Elastic Email policy insists on an additional UNSUBSCRIBE link at the bottom of each sent email. If this is a problem for your needs, consider using other providers. [See discussion here](https://meta.discourse.org/t/remove-or-merge-elastic-email-unsubscribe/70236).
+NOTE: By default, Elastic Email will add an additional UNSUBSCRIBE link at the bottom of each sent email. You need to work with them to [disable that link](https://meta.discourse.org/t/remove-or-merge-elastic-email-unsubscribe/70236/39?u=dandv), so that Discourse users can manage their subscription through Discourse.
```

Comment:
```suggestion
NOTE: By default, Elastic Email will add an additional UNSUBSCRIBE link at the bottom of each sent email. You need to work with them to [disable that link](https://meta.discourse.org/t/remove-or-merge-elastic-email-unsubscribe/70236/39), so that Discourse users can manage their subscription through Discourse.
```